composables/drop/useGenerativeDropMint.ts

```diff
@@ -26,6 +26,7 @@ export type UnlockableCollectionById = {
 export function useCollectionEntity(collectionId?: string) {
   const { drop } = useDrop()
   const { client } = usePrefix()
+  const { accountId } = useAuth()
```

Comment:
Why do you need account? 

What if account is undefined / user not logged in

Reply:
> Why do you need account?

from this query we get the data that we later use  for `mintedAmountForCurrentUser` and that's why need the accountId


![CleanShot 2024-04-15 at 11 59 17@2x](https://github.com/kodadot/nft-gallery/assets/44554284/ab569552-695d-426d-88be-1bd8c1d5a3de)


![CleanShot 2024-04-15 at 11 58 42@2x](https://github.com/kodadot/nft-gallery/assets/44554284/dd50eb61-3a04-4ade-b5ed-0ad88dee1fe0)


> What if account is undefined / user not logged in

accountId will be '' and 0 will be returned. 

![CleanShot 2024-04-15 at 12 01 00@2x](https://github.com/kodadot/nft-gallery/assets/44554284/93824a7c-a8bb-4751-b9f7-9de230db07c1)